src/main/java/mate.academy/service/FileReaderService.java

```diff
@@ -1,7 +1,9 @@
 package mate.academy.service;
 
 import java.util.List;
+import mate.academy.lib.Component;
 
+@Component
```

Comment:
The `@Component` annotation is typically used for classes that are meant to be instantiated and managed by a dependency injection framework. Applying it to an interface is not appropriate. Consider removing the `@Component` annotation from the interface.

Reply:
changes have been made